rules/mdx-vs-markdown/rule.md

```diff
@@ -0,0 +1,63 @@
+---
+type: rule
+title: Do You Know When to Use MDX?
+seoDescription: " Learn when to use MDX for custom components and advanced
+  functionality, regardless of the framework, versus sticking with Markdown for
+  simplicity."
+uri: mdx-vs-markdown
+authors:
+  - title: Jack Pettit
+    uri: https://ssw.com.au/people/jack-pettit
+guid: 9231556d-6d72-43b8-ba7d-ce704e64221c
+related: []
+redirects: []
+---
+Let's face it, not all content is created equal. Sometimes you just need a simple document, and other times you want something more dynamic and interactive.
+
+<!--endintro-->
+
+
+## When to Use Markdown (.md)
+
+Markdown is perfect for straightforward content. Think of it like writing a clear, no-frills document. You'll want to use Markdown when:
+
+- You're creating something simple like a blog post, documentation, or guide
+- Your team includes people who aren't tech experts
+- You want your page to load quickly
+- You just need basic formatting like headings, lists, and images
+
+::: greybox
+**Example:** A recipe blog post with some text, headings, and a few pictures. Markdown handles this beautifully without any extra complexity.
+:::
+
+## When to Use MDX (.mdx)
+
+MDX steps up when you need something more powerful. It lets you add interactive elements and custom components to your content. You'll want MDX when:
+
+- You need interactive features that go beyond static text
+- You want to include custom components from different web frameworks
+- Your content requires some programming logic
+- You're creating tutorial content with live examples
+
+
+::: greybox
+**Example:** A coding tutorial with an interactive chart showing performance metrics, or a documentation page with a live code editor where readers can try out code in real-time.
+::: 
+
+## Things to Consider
+
+MDX isn't perfect for every situation. Before you jump in, consider:
+
+- **Complexity**: It's more advanced than plain Markdown. Your non-technical team might find it tricky
+- **Performance**: Too many fancy components can slow down your page
+- **Extra Setup**: You'll need to manage more technical dependencies
+
+## The Golden Rule
+
+Choose Markdown for simple, fast content. Choose MDX when you need more interactive and dynamic features.
+
+The key is to start simple. Use Markdown for most of your content, and only switch to MDX when you truly need those extra capabilities.
```

Comment:
@JackDevAU 

These last 2 sentences (1 being in a box) are too similar, they prety much say the same thing.

- [ ] 1. Please keep only one. I personally prefer the second sentence, but I would leave it as normal paragraph, outside the info box

Reply:
✅ Done - Removed the info box